lib/neo4j/migration.rb

```diff
@@ -123,5 +126,68 @@ def new_id_for(model)
         end
       end
     end
+
+    class RelabelRelationships < Neo4j::Migration
+      attr_accessor :relationships_filename, :old_format, :new_format
+
+      def initialize(path = default_path)
+        @relationships_filename = File.join(joined_path(path), 'relabel_relationships.yml')
+      end
+
+      MESSAGE = <<MESSAGE
+# Provide relationships which should be relabled.
+# relationships: [students,lessons,teachers,exams]\nrelationships: []
+# Provide old and new label formats:
+# Allowed options are lower_hashtag, lower, or upper
+formats:\n  old: lower_hashtag\n  new: lower
+MESSAGE
```

Comment:
Would be nice to see a full example of this `yml`.  Perhaps in the docs.  The docs are under the `docs` directory and they generate this site: http://neo4jrb.readthedocs.org/en/7.0.x/